src/main/java/com/thealgorithms/maths/FibonacciNumber.java

```diff
@@ -0,0 +1,35 @@
+// Explanation:- https://en.wikipedia.org/wiki/Fibonacci_sequence#Binet's_formula
+
+import java.math.BigInteger;
+
+public final class FibonacciNumber {
+    private FibonacciNumber() {
+    }
+
+    // Compute the limit for n that fits in a long
+    // Reducing the limit to 70 due to incorrect results for larger inputs
```

Comment:
Please provide a _better description_ - it is a good idea to mention _floating point arithmetic errors_.